CoreTableView/src/au/gov/asd/tac/constellation/views/tableview/factory/TableViewPageFactory.java

```diff
@@ -117,11 +117,11 @@ public Node call(Integer pageIndex) {
         if (allTableRows != null) {
 
             // This action only effects selected only mode variables but runs no
-            // matter if the table is in selected only mode or not.
+            // matter if the table is in selected only mode or not.pluginExecution(Java) "==" and "!=" should not be used when "equals" is overridden
```

Comment:
the addition here could probably be removed

Reply:
I think this was a rogue CTRL+V thanks for catching